apps/docs/content/components/UnorderedList.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Unordered List
+status: Experimental
+figma: 'https://www.figma.com/file/BJ95AjraesmRCWsKA013GS/branch/NzBMPIfPrFU9I87d3k7KVq/Primer-Brand?node-id=4045%3A17093&t=XJ9W58VgtbVo74gb-0'
+source: https://github.com/primer/brand/tree/main/src/List/UnorderedList/UnorderedList.tsx
+storybook: '/brand/storybook/?path=/story/components-unorderedlist--default'
+description: Use the UnorderedList component to render a list of unordered items.
+---
+
+```js
+import {UnorderedList} from '@primer/react-brand'
+```
+
+## Examples
+
+### Default
+
+```jsx live
+<UnorderedList variant="default">
```

Comment:
Nit. Do you need variant="default" prop here if it's set as a default value?

Reply:
Yes because not setting the variant would default to nothing which would in turn be numeric. The variant displays the leading visual for a default list. Had these all been one component the logic behind these would have been simpler. And easier for a user to use.

Reply:
Updated in next push